crates/interfaces/src/p2p/headers/client.rs

```diff
@@ -23,6 +23,14 @@ pub trait HeadersClient: DownloadClient {
     /// Sends the header request to the p2p network and returns the header response received from a
     /// peer.
     async fn get_headers(&self, request: HeadersRequest) -> PeerRequestResult<BlockHeaders>;
```

Comment:
I guess we can add a default impl here to call with Normal

Reply:
Does this work?
```Rust
 async fn get_headers(&self, request: HeadersRequest) -> PeerRequestResult<BlockHeaders> {
    self.get_headers_with_priority(request, Priority::Normal).await
 }
```

Reply:
This means we'll have only get_headers_with_priority() in the ```impl HeadersClient ...```

Reply:
yes this should work I think